extensions/wikia/CommunityPage/styles/entrypoint/EntryPoint.scss

```diff
@@ -37,7 +23,9 @@ $logo-url: $logo-url-light;
 	text-align: center;
 
 	.community-page-entry-point-logo {
-		background-image: url($logo-url); /* inline */
+		background: $color-white url($logo-url) 50% 50%/18px 18px no-repeat; /* inline */
+		border: 1px solid $color-page-border;
+		border-radius: 15px;
 		height: 30px;
```

Comment:
would it make sense to move border-radius, height & width to a `$entry-point-logo-size: 30px;` ?